segbot_sensors/launch/diagnostic/diagnostics.launch

```diff
@@ -1,8 +1,9 @@
 <launch>
 
   <!-- Set necessary parameters -->
-  <param name="email_alert_outbound" type="string" value="maxsvetlik@gmail.com" />
-  <param name="email_alert_sender" type="string" value="bwi@cs.utexas.edu" />
+  <param name="email_alert_outbound" type="string" value="maxsvetlik@utexas.edu matteo@cs.utexas.edu shiqi.zhang6@gmail.com 
+jsinapov@cs.utexas.edu pstone@cs.utexas.edu piyushk@gmail.com" />
```

Comment:
When we get the chance, let's make a new BWI mailing list, specifically for these messages. That way, people can subscribe and unsubscribe without having to modify this code and update all the robots. Let's name it something like ``bwi-operators@cs.utexas.edu``.